src/core/components/UserChip/styles.js

```diff
@@ -32,6 +32,7 @@ const Close = styled(Remove)`
 const RoundButton = styled(Button)`
   padding: 0.4rem 0.55rem;
   border-radius: 10rem;
+  height: 30px;
```

Comment:
not for this pr in particular, but what do you think about defining some constants for dimensions, so we can have things proportional at least?

Reply:
In WAX component library, we have it defined as spacing. And we will reuse it here, eventually.